src/ui/templates/search/autocomplete.hbs

```diff
@@ -1,4 +1,21 @@
 <div class="yxt-AutoComplete-wrapper js-yxt-AutoComplete-wrapper" aria-expanded="{{hasResults}}">
+  <span class="yxt-AutoComplete-resultsCount sr-only" aria-live="assertive">
+    {{#unless beforeSearch}}
+      {{#if hasResults}}
+        {{#each sections}}
+          {{translate
+            phrase='[[resultsCount]] [[label]] autocomplete option found.'
+            pluralForm='[[resultsCount]] [[label]] autocomplete options found.'
+            resultsCount=resultsCount
+            label=label
+            count=resultsCount
+          }}
+        {{/each}}
+      {{else}}
+        {{translate phrase='0 autocomplete options found.'}}
```

Comment:
Do we need to special case this? Or can it be handled by the more generic '[resultsCount]] autocomplete options found' we will have above?

Reply:
Understood. But do we need a specific, new phrase in this case? Could we not use 
```
            {{translate
              phrase='[[resultsCount]] autocomplete option found.'
              pluralForm='[[resultsCount]] autocomplete options found.'
              resultsCount=resultsCount
              count=resultsCount
            }}
```
here too? That would save us having to have an additional translation in Smartling just for this.

Reply:
The `resultCount` would be 0 here.